graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -102,3 +114,17 @@ def _get_vertex_type_from_sqlalchemy_table(table):
         if maybe_property_type is not None:
             properties[name] = PropertyDescriptor(maybe_property_type, default)
     return VertexType(table.name, False, properties, {})
+
+
+def _get_edge_types(tables):
+    """Return the EdgeType objects corresponding to SQLAlchemy ForeignKey objects."""
+    edge_types = []
+    for table in six.itervalues(tables):
+        for column in table.get_children():
+            for foreign_key in column.foreign_keys:
```

Comment:
Is this multiple foreign keys on the same column?

Reply:
Yes. I just realized that because of this the edge key names are not unique even if we prepend the table name to the column name. So I guess we have two options. One is to prepend the table name and append the outgoing table name to the EdgeTyoe. The other is to not allow columns to have multiple foreign keys.

Reply:
(had the conversation offline) Makes more sense to not allow multiple foreign keys on the same column.